backend-rust/src/connection.rs

```diff
@@ -69,17 +69,20 @@ impl ConnectionCursor for i64 {
 /// Cursor Connection resolver.
 pub struct ConnectionQuery<A> {
     /// The lower to use for the SQL query.
-    pub from:  A,
+    pub from:       A,
     /// The upper to use for the SQL query.
-    pub to:    A,
+    pub to:         A,
     /// The limit to use for the SQL query.
-    pub limit: i64,
+    pub limit:      i64,
     /// If the `last` elements are requested instead of the `first` elements
     /// (indicated by the `last` key being set when creating a new
     /// `ConnectionQuery`), the edges/nodes should be ordered in reverse
     /// (DESC) order before applying the range. This allows the range from
     /// `from` to `to` to be applied starting from the last element.
-    pub desc:  bool,
+    /// TODO: Migrate away from this and use descending instead
+    pub desc:       bool,
+    /// Reflects whether the query should be descending or ascending
+    pub descending: bool,
```

Comment:
This is not about the connection itself being descending, but rather the subquery should be descending or ascending based on whether last/first was provided. The connection page can be descending while this field is true.